src/index.html

```diff
@@ -1,13 +1,43 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body class="page-content">
+
+    <div class="product-card" data-qa="card">
+      <img 
+        class="product-card__img" 
+        src="./images/imac.jpeg" 
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <h2 class="product-card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <p class="product-card__code">Product code: 195434</p>
+
+      <div class="product-card__rate">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p>Reviews: 5</p>
+      </div>
+
+      <p class="product-card__price-text">Price:</p>
+      
+      <p class="product-card__price-value">$2,199</p>
+
+      <a class="product-card__link" data-qa="hover">Buy</a>
```

Comment:
```suggestion
      <p class="product-card__price-text">Price:</p>
      <p class="product-card__price-value">$2,199</p>
      <a class="product-card__link" data-qa="hover">Buy</a>
```

There is no need to use empty lines between single-line siblings.